tests/test_dewpoint.py

```diff
@@ -1,127 +1,209 @@
+# (C) Copyright 2025 Anemoi contributors.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
 import numpy as np
+import pytest
+from anemoi.utils.testing import skip_if_offline
 
 from anemoi.transform.filters import filter_registry
 from anemoi.transform.sources import source_registry
-from anemoi.transform.testing import ListSource
-from anemoi.transform.testing import convert_to_ekd_fieldlist
+from anemoi.transform.testing import SelectFieldSource
+
+from .utils import assert_fields_equal
+from .utils import collect_fields_by_param
 
-prototype = {
+MOCK_FIELD_METADATA = {
```

Comment:
If the `MOCK_FIELD_METADATA` can be reused across some test, could we move it to utils?

Reply:
I was thinking we could have the current 'MOCK_FIELD_METADATA' as  a template on utils to make it more accessible for people to reuse across all the tests consistently and then they could reuse that overwriting it on the particular test. But I see that by having it in all tests, it's decoupled so easier to modify and run the tests. No strong preferences, we can keep it like this for now